scout/server/blueprints/cases/controllers.py

```diff
@@ -87,7 +89,9 @@ def coverage_report_contents(base_url, institute_obj, case_obj):
     request_data["level"] = institute_obj.get("coverage_cutoff", 15)
 
     # Collect the coverage report HTML string
-    resp = requests.post(base_url + "reports/report", data=request_data)
+    resp = requests.post(
+        base_url + "reports/report", timeout=COVERAGE_REPORT_TIMEOUT, data=request_data
```

Comment:
try:
    requests.post(url, data=payload, timeout=5)
except requests.Timeout:
  something?

Reply:
It might crash otherwise

Reply:
Does indeed; I didn't realise it would raise rather than just silently time out. Nicer like this anyway!